### ISSUE TITLE
Automated ExUnit based test executor

### ISSUE THREAD

**Issue by maennchen** [erlef/oidcc_conformance#3]
Should create testplans automatically and execute all tests.

Needs an API token.

Browser to be controlled using Wallaby.

Reference: https://gitlab.com/paulswartz/ueberauth_oidcc_certification

**Comment by maennchen** [comment]
BLocked by  #2

**Comment by maennchen** [comment]
@paulswartz Since the GitLab repo does not have a LICENSE: Can you quickly comment that you agree with this usage of your code?

**Comment by paulswartz** [comment]
Yep, this is fine! And I'll add a LICENSE when I'm back at a computer.
